capstone/capdb/models.py

```diff
@@ -970,9 +1033,26 @@ def update_frontend_urls(cls, cite_strs, update_elasticsearch=True, batch_size=1
                     cls.reindex_cases(to_update)
 
     @classmethod
-    def reindex_cases(cls, cases):
+    def reindex_cases(cls, cases, prune_duplicates=False):
```

Comment:
Having `reindex_cases` remove out of scope cases is a great idea! Can we make it happen all the time, and depend on case.in_scope instead of on case.duplicate?

Reply:
Yep.